Minigames/src/main/java/au/com/mineauz/minigames/commands/DebugCommand.java

```diff
@@ -52,18 +64,109 @@ public String getPermission() {
     @Override
     public boolean onCommand(CommandSender sender, Minigame minigame,
                              String label, String[] args) {
-        Minigames.getPlugin().toggleDebug();
-        if (Minigames.getPlugin().isDebugging())
-            sender.sendMessage(ChatColor.GRAY + "Debug mode active.");
-        else
-            sender.sendMessage(ChatColor.GRAY + "Deactivated debug mode.");
-        return true;
+        if(args.length > 1) {
+            switch (args[0].toUpperCase()){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Switch statements should have a default label](https://app.codacy.com/manual/Narimm/Minigames/pullRequest?prid=5635375)